### PR TITLE
SNI-4875

### PR DESCRIPTION
**Pull request by tomxelliott**
### Change description ###
Migrate caseworker contact parties from Release 3 branch into master

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SNI-4875

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/ContactPartiesSelectDocument.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import uk.gov.hmcts.sptribs.ciccase.model.State;
+import uk.gov.hmcts.sptribs.common.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.sptribs.common.ccd.PageBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ContactPartiesSelectDocument implements CcdPageConfiguration {
+
+    private static final int MAX_DOCUMENT_COUNT = 10;
```
[the remark this answers is not in the document]
Comment (tomxelliott) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/ContactPartiesSelectDocument.java:17]: MAX_DOCUMENT_COUNT has different values in different files (some 10, some 5) so cannot be added to config file as it is not the same throughout the app

**Review by rajaram-nagarajan — DISMISSED** [review]
LGTM

**Review by osheaf01 — APPROVED** [review]
LGTM

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by JakeCG — APPROVED** [review]

**Pull request merged**